server/sonar-server/src/main/java/org/sonar/server/platform/platformlevel/PlatformLevel4.java

```diff
@@ -498,6 +498,7 @@ public PlatformLevel configure() {
       org.sonar.server.usergroups.ws.SearchAction.class,
       org.sonar.server.usergroups.ws.CreateAction.class,
       org.sonar.server.usergroups.ws.DeleteAction.class,
+      org.sonar.server.usergroups.ws.UpdateAction.class,
```

Comment:
Well, it was not available at the time I opened the PR, but why not give it a shot.